tck/src/test/java/org/eclipse/microprofile/jwt/tck/container/jaxrs/ClaimValueInjectionTest.java

```diff
@@ -431,5 +431,19 @@ public void verifyInjectedCustomDouble() throws Exception {
         Assert.assertTrue(reply.getBoolean("pass"), reply.getString("msg"));
     }
 
-
+    @RunAsClient
+    @Test(groups = TEST_GROUP_CDI,
+          description = "Verify that the injected customDouble claim is as expected")
```

Comment:
Wrong description, refers to Double but it is testing Boolean.

`String uri = baseURL.toExternalForm() + "endp/verifyInjectedCustomBoolean";`